check_signed_off.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Checking range: ${TRAVIS_COMMIT_RANGE}"
+
+NEW_COMMITS=( $(git log  --format=format:%H "$TRAVIS_COMMIT_RANGE") )
```

Comment:
Can be simplified by using `git rev-list`. Needs no arguments other than the range. :-)